README.md

```diff
@@ -1,14 +1,25 @@
-Setup:
+# Welcome to the Corner Club
+
+Good vibes are implicit. Good code is essential.
+
+### How to configure the project for local testing:
 1) Open the solution in Visual Studio.
 2) Go to Tools>Nuget Package Manager>Package Manager Console
 3) In the console at the bottom of the screen type--> update-database
 4) In the command prompt enter--> pip install pyodbc
 5) Run insertData.py
 6) Click IIS Express in Visual Studio to run the site
 
-Notes:
+### How to configure MySQL Workbench with Azure:
+
```

Comment:
I thought we were using Microsoft SQL Server?

Reply:
whoopsie pie, gonna close this issue